.github/CODEOWNERS

```diff
@@ -0,0 +1,23 @@
+# See for instructions on this file https://help.github.com/articles/about-codeowners/
+
+*help.py @sptramer
+
+/tools/ @troydai @derekbekoe
+/scripts/ @troydai @derekbekoe
+/packaged_releases/ @derekbekoe
+/src/azure-cli-testsdk/ @troydai
+/src/command_modules/azure-cli-acr/ @djyou
+/src/command_modules/azure-cli-acs/ @rjtsdl
+/src/command_modules/azure-cli-appservice/ @yugangw-msft
+/src/command_modules/azure-cli-backup/ @dragonfly91
+/src/command_modules/azure-cli-batch/ @annatisch 
+/src/command_modules/azure-cli-cloud/ @derekbekoe
+/src/command_modules/azure-cli-container/ @yolo3301
+/src/command_modules/azure-cli-eventgrid/ @kalyanaj
```

Comment:
Cosmosdb should be @dmakwana
Interactive @williexu (?)
Monitor @troydai 
IOT ... ?
RDBMS ... ?
Storage @troydai (?)
resource @brendandburns (?)
Datalake ... ?
DTL ... ?
CDN  @tjprescott

Reply:
I added the ones you didn't put ? around.
We can add more later depending on if this works well.